### PR TITLE
docs/x509: fix verification example

### PR DESCRIPTION
**Pull request by woodruffw**
Fixes a few of my own mistakes in this example, and makes it complete (with a peer and intermediates).

(Manually ran it locally and confirmed that it works, as well.)

### CODE REVIEW

File: docs/x509/verification.rst [hunk 1]
```diff
@@ -17,15 +17,63 @@ the root of trust:
 
 .. code-block:: python
```
Comment (woodruffw) [inline docs/x509/verification.rst:18]: I think I could make this into `.. doctest::` instead, if desired.

File: docs/x509/verification.rst [hunk 2]
```diff
@@ -17,15 +17,63 @@ the root of trust:
...
 
-    from cryptography.x509 import Certificate, DNSName, load_pem_x509_certificates
+    from cryptography.x509 import Certificate, DNSName, load_pem_x509_certificate, load_pem_x509_certificates
     from cryptography.x509.verification import PolicyBuilder, Store
     import certifi
 
+    from datetime import datetime
```
Comment (alex) [inline docs/x509/verification.rst:24]: Hmm, this makes it super verbose. Is there some way for us to make clear what perr and untrusted are without making the example 200 lines long

Comment (woodruffw) [reply docs/x509/verification.rst:24]: If you're okay with it going back to being non-functional, I could probably do something like `peer, intermediates = load_pem_x509_certificates(open("whatever.pem").read())`. WDYT?

Comment (alex) [reply docs/x509/verification.rst:24]: Hmm, so using `doctest` + `testsetup` and then adding a comment explaining what they are would be my suggestion

**Review by alex — APPROVED** [review]

**Pull request merged**